File: 01_multof3and5.py
# Solution to https://projecteuler.net/problem=1
t=0
n=1
for x in 3,5:
    n = 1
    while (x*n < 1000):
        print("x*n",x*n)
        t+=x*n
        print("t",t)
        n+=1
# remove powers of 15
n=1
x=15
while (x*n < 1000):
    print("-x*n",x*n)
    t-=x*n
    n+=1
    print("t",t)

File: 01_multof3and5.js
// Solution to https://projecteuler.net/problem=1
console.log(
    Array.apply(null,{length : 999})
        .map((v,i)=>(i+1))
            .filter(v=>(v%3 == 0) || (v%5 == 0))
                .reduce((sum, v) => sum += v))
